src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/IPhysicalProjectTreeStorage.cs

```diff
@@ -14,11 +14,54 @@ namespace Microsoft.VisualStudio.ProjectSystem
     /// </summary>
     /// <remarks>
     ///     This interface provides a simple facade over the <see cref="IFileSystem"/>, <see cref="IFolderManager"/>, 
-    ///     <see cref="IProjectTreeService"/> and <see cref="IProjectItemProvider"/> interfaces.
+    ///     <see cref="IProjectTreeService"/>, <see cref="IProjectItemProvider"/> and <see cref="IProjectItemProvider"/> interfaces.
     /// </remarks>
     [ProjectSystemContract(ProjectSystemContractScope.UnconfiguredProject, ProjectSystemContractProvider.Private, Cardinality = ImportCardinality.ExactlyOne)]
     internal interface IPhysicalProjectTreeStorage
     {
+        /// <summary>
+        ///     Creates a zeroed file on disk, adding it add to the physical project tree.
```

Comment:
Zeroed or empty? To me a zeroed file has non-zero length and contains all zeroes.

Reply:
Yep, good call.